CONTRIBUTING.md

```diff
@@ -5,5 +5,6 @@ Implement more OpenCV functions/modules and submit a PR request.
 Before submitting a PR, make sure you have formatted both C/C++ code and Rust
 code, otherwise Travis will complain.
 
-- For Rust, run `cargo fmt`
-- For C/C++, run `clang-format -i native/*`
+You have to have installed `llvm` ([download link](http://releases.llvm.org/download.html)) to make formatting happen.
```

Comment:
We should be concrete to mention `clang-format`, https://clang.llvm.org/docs/ClangFormat.html, instead of just giving a pointer to `llvm`. Users don't need the whole LLVM tool chain.

And I think on Mac, you can simply do `brew install clang-format`.

Reply:
Sry, I wasn't aware that you shouldn't have entire toolchain on *nix. Going to fix it.